### DISCUSSION TITLE
Mettre la bannière à gauche

### DISCUSSION THREAD

**Discussion by dominic33** [AmauriC/tarteaucitron.js#681] [Ideas]
Bonjour,
est il possible d'afficher la bannière à gauche car la droite et souvent occupé par Tchat.
Comme cela est possible pour l'icone
Par avonce merci,
Cordialement
Dc

**Comment by AmauriC** [comment]
Avec ce CSS:
```
div#tarteaucitronAlertSmall {
    right: auto!important;
    left: 0;
}
```

**Comment by dominic33** [comment]
Merci c'est nickel

bien cordialement
Dc
